.gitlab-ci/docker-ros.yml

```diff
@@ -71,19 +80,20 @@ stages:
   - Build dev Images
   - Build run Images
   - Test ROS Industrial CI
+  - Slim Images
   - Push Multi-Arch Images
 
 
 default:
-  image: docker:20.10.22-git
+  image: docker:20.10.22-git # TODO: upgrade?
```

Comment:
upgrade?

Reply:
I would suggest to test and upgrade this within a separate PR.